src/components/alfresco/AlfrescoForm.jsx

```diff
@@ -74,7 +74,7 @@ class AlfrescoForm extends Component {
         const component = this.props.properties.filter(property => property.name === containerKey)[0];
 
         return (
-          <Grid key={containerIndex} item xs={12}>
+          <Grid item key={containerIndex} xs={12} style={{ marginBottom: 24 }}>
```

Comment:
can we move the inline styles to classes?

Reply:
Yes, good point.